runtime/src/iree/hal/utils/fd_file.c

```diff
@@ -319,8 +319,7 @@ static iree_status_t iree_hal_fd_file_read(iree_hal_file_t* base_file,
   if (iree_status_is_ok(status) &&
       !iree_all_bits_set(iree_hal_buffer_memory_type(buffer),
                          IREE_HAL_MEMORY_TYPE_HOST_COHERENT)) {
-    status =
-        iree_hal_buffer_mapping_flush_range(&mapping, buffer_offset, length);
+    status = iree_hal_buffer_mapping_flush_range(&mapping, 0, length);
```

Comment:
Does that mean at line 304, buffer_offset need to be replaced with 0? and this function does not the input parameter buffer_offset anymore?

Reply:
Nope, you can think of a mapping as a view into the buffer. When, so we want to map a bunch of bytes of the buffer into some host-visible memory, the offset on 304 from the start of the buffer that we want to map. But when we want to flush (or invalidate) the offset is the offset in the mapping.